pages/governance/slashing_a_validator.md

```diff
@@ -0,0 +1,44 @@
+# Slashing a validator
+
+The chain supports slashing of misbehaving validators through governance vote.
+
+## Proposal Message
+
+The proposal should contain a single [MsgSlashValiator](https://github.com/dydxprotocol/v4-chain/blob/protocol/v4.0.0/proto/dydxprotocol/govplus/tx.proto#L19) message for each validator that should be slashed. See the inline comments for details on each field.
+
+**💡 Important: Both `tokens_at_infraction_height` and `slash_factor` must be set correctly, otherwise undelegations and redelegations might be slashed disproportionately to the rest of the validator's stake.**
+
+### Example Proposal Json
+
+Below is an example proposal JSON file to propose a slashing a validator's stake by 0.2. After execution the validator will lose 20% of their `tokens_at_infraction_height`. 
+```json
+{
+    "title": "Slash a validator",
+    "summary": "We are proposing to slash this misbehaving validator for X reasons.",
+    "deposit": "10000000000000000000000adv4tnt",
+    "messages": [
+        {
+          "@type": "/dydxprotocol.govplus.MsgSlashValidator",
+          "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+          "validator_address": "dydxvalcons1z79h40nmd777scs93qjxaeak8m2cl6hpqg2rx9",
+          "slash_factor": "0.2",
+          "tokens_at_infraction_height": "500000000000000000000000",
+          "infraction_height": 7647362
+        }
+    ]
+}
+  ```
+
+## Submitting an Onchain Proposal
+
+Follow instructions [here](./submitting_a_proposal.md) to submit an onchain proposal.
+
+
+#### Disclaimer
+Users considering using the permissionless markets function of the dYdx v4 software are 
```

Comment:
```suggestion
Users considering using the permissionless markets function of the dYdX v4 software are 
```

Reply:
Why is this paragraph relevant for this page?